recipes-go/cpu/cpu.bb

```diff
@@ -0,0 +1,13 @@
+DESCRIPTION = "cpu command in Go, inspired by the Plan 9 cpu command"
+
```

Comment:
@PLangowski please add a HOMEPAGE of used project https://docs.yoctoproject.org/singleindex.html#term-HOMEPAGE

Reply:
Done f64259079b3fe8c4fba7c5b611fd0f26c7167182